README.md

```diff
@@ -86,12 +127,12 @@ bases:
   - github.com/banzaicloud/istio-operator/config/overlays/auth-proxy-enabled?ref=release-1.6
 ```
 
-### Installation with Backyards
+### Installation with [Backyards](https://banzaicloud.com/products/backyards/)
 
-Go grab and install the [Backyards CLI](https://github.com/banzaicloud/backyards-cli) tool.
+Go grab and install Istio with the [Backyards CLI](https://github.com/banzaicloud/backyards-cli) tool.
```

Comment:
I'd rather skip the command in this section, and write something like:
To install Istio with the Backyards CLI tool, see the [Backyards Quick Start Guide](https://banzaicloud.com/docs/backyards/quickstart/).